.github/workflows/test-on-centos.yml

```diff
@@ -0,0 +1,42 @@
+name: Run tests in Centos container
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 1 * * *'
+
+jobs:
+  tests-on-centos:
+    strategy:
+      matrix:
+        version: ["c8s", "c9s"]
+    name: "Unittests on Centos ${{ matrix.version }}"
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v4
+      - name: "Run in container"
+        uses: addnab/docker-run-action@v3
+        with:
+          image: quay.io/osbuild/osbuild-ci-${{ matrix.version }}
+          options: --privileged -v ${{ github.workspace }}:/osbuild
+          run: |
+            cd /osbuild
+            if [ "${{ matrix.version }}" = "c8s" ]; then
```

Comment:
nitpick: this condition can be eliminated by defining the `matrix` items as objects with multiple properties (see https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#example-using-a-multi-dimension-matrix). The `-k` filter string would be the additional property and it would be different per CS version. This way, the actual test execution could be done in the same way regardless of the CS version.

Reply:
Done